Content.Shared/GameObjects/Components/Damage/DamageableComponent.cs

```diff
@@ -354,7 +354,7 @@ public void Heal()
                     healThisCycle = 0;
 
                     int healPerType;
-                    if (healingLeft > -types.Count)
+                    if (healingLeft < types.Count)
                     {
                         // Say we were to distribute 2 healing between 3
                         // this will distribute 1 to each (and stop after 2 are given)
```

Comment:
Github's not letting me comment on it but the line below of healPerType should be 1 given healingLeft is now positive.

Reply:
Fixed.